src/textual/widgets/_collapsible.py

```diff
@@ -99,6 +99,26 @@ class Collapsible(Widget):
     }
     """
 
+    class Toggled(Message):
+        """Event sent when the `Collapsible` is toggled.
+
+        Can be handled using `on_collapsible_toggled` in a subclass of
+        [`Collapsible`][textual.widgets.Collapsible] or in a parent widget in the DOM.
+        """
+
+        def __init__(self, collapsible: Collapsible) -> None:
+            self.collapsible: Collapsible = collapsible
+            """The collapsible that was pressed."""
+            super().__init__()
+
+        @property
+        def control(self) -> Collapsible:
+            """An alias for [Toggled.collapsible][textual.widgets.Collapsible.Toggled.collapsible].
+
+            This will be the same value as [Toggled.collapsible][textual.widgets.Collapsible.Toggled.collapsible].
+            """
```

Comment:
I copied the docstring from `Button.Pressed`.
Doesn't this feel a bit redundant, though?